Tasks/ManualValidationV0/task.json

```diff
@@ -0,0 +1,60 @@
+{
+    "id": "2003AD3A-104A-451A-9238-60474AB294E6",
+    "name": "ManualValidation",
+    "friendlyName": "Manual validation",
+    "description": "Pause deployment and wait for manual validation",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/manual-validation",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=870234)",
+    "category": "Deploy",
+    "visibility": [
+        "Release"
+    ],
+    "author": "Microsoft Corporation",
+    "runsOn": [
+        "Server"
+    ],
+    "version": {
+        "Major": 0,
+        "Minor": 0,
+        "Patch": 1
+    },
+    "inputs": [
+        {
+            "name": "validators",
+            "type": "identities",
+            "label": "Manual Validators",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "Only users added in validators list can act on a manual validation."
```

Comment:
We need to be sure of this also . In MI , there is a separate Security bit which gives ability to be able to Intervene , and this is for notification purpose only. Can we confirm once just .

Reply:
This field is only to specify who we send emails to. I have corrected the description